pkg/dashboard/list.js

```diff
@@ -55,24 +57,26 @@ var common_plot_options = {
 
 var resource_monitors = [
     { selector: "#dashboard-plot-0",
-      plot: {
-          direct: [
-              "kernel.all.cpu.nice",
-              "kernel.all.cpu.user",
-              "kernel.all.cpu.sys"
-          ],
-          internal: [
-              "cpu.basic.nice",
-              "cpu.basic.user",
-              "cpu.basic.system"
-          ],
-          units: "millisec",
-          derive: "rate",
-          factor: 0.1  // millisec / sec -> percent
+      plot: function (machine_info) {
+          return {
+              direct: [
+                  "kernel.all.cpu.nice",
+                  "kernel.all.cpu.user",
+                  "kernel.all.cpu.sys"
+              ],
+              internal: [
+                  "cpu.basic.nice",
+                  "cpu.basic.user",
+                  "cpu.basic.system"
+              ],
+              units: "millisec",
+              derive: "rate",
+              factor: 0.1 / machine_info.cpus // millisec / sec -> percent
```

Comment:
Will that work initially while the `machine_info` (as in the imported function) promise isn't resolved yet and the passed `machine_info` (local parameter) here is still `{}`? `0.1/undefined == NaN`, so that won't cause an exception by itself, just want to make sure that something within the plot does not get upset about it.

Reply:
> Will that work initially while the machine_info (as in the imported function) promise isn't resolved yet and the passed machine_info (local parameter) here is still {}? 

This will never happen, right?  `plot_add` is only called for a given address once the promise is resolved.